java/code/src/com/redhat/rhn/manager/contentmgmt/test/ContentManagerTest.java

```diff
@@ -1051,6 +1051,46 @@ public void testBuildProjectWithFilters() throws Exception {
         assertTrue(cp.getProjectFilters().isEmpty());
     }
 
+    public void testBuildProjectWithNevrFilters() throws Exception {
```

Comment:
I like when, for test methods, I see just one/two lines of comment at the beginning that will explain me briefly and clearly what the test method does, in terms of *describing the tested scenario* or something like that. In this case I think, as an idea, something like `Make sure the package set to be filtered out by its *nevr* is really filtered out in the output built. The check is made by counting the number of packages in the target destination.`

Reply:
I agree only partially: I agree that it's a good thing to document code, but not "too much". 1, When the underlying code changes in the future, it's quite likely that the doc is forgotten. 2, Moreover: the documentation in tests is for other developers, so I'd suppose they could just look at the `asserts`, so that statement `The check is made by counting the number of packages in the target destination.` would be obvious to them.

Reply:
Agreed. I think it's only a personal feeling, but yeah, avoiding adding documentation that surely goes out of sync at some point (and I am pretty sure it will) is a stronger argument :wink: